SSDynamicText/SSDynamicButton.m

```diff
@@ -20,6 +23,23 @@ - (void)setup;
 
 @implementation SSDynamicButton
 
+- (NSMutableDictionary *)baseAttributedTitlesDictionary {
+    if (_baseAttributedTitlesDictionary == nil) {
+        _baseAttributedTitlesDictionary = [NSMutableDictionary dictionary];
+    }
+    return _baseAttributedTitlesDictionary;
+}
+
+- (void)setAttributedTitle:(NSAttributedString *)title forState:(UIControlState)state {
+    NSNumber *key = @(state);
+    if (title) {
+        [self.baseAttributedTitlesDictionary setObject:title forKey:key];
```

Comment:
Well, in else statement i have to use `-removeObjectForKey` and for me this code looks better with `setObject:forKey`